### PR TITLE
Implemented change password

### PR DESCRIPTION
**Pull request by SundeepChand**
Migrated [https://gitlab.com/SUSE-UIUX/eos-user-story/-/merge_requests/111](https://gitlab.com/SUSE-UIUX/eos-user-story/-/merge_requests/111)

Goes with [https://github.com/EOS-uiux-Solutions/strapi/pull/3](https://github.com/EOS-uiux-Solutions/strapi/pull/3)

### CODE REVIEW

File: src/hooks/useAuth.js [hunk 1]
```diff
@@ -98,12 +98,40 @@ const useAuth = () => {
...
+      .catch((err) => {
+        if (err.message === 'Network Error')
+          dispatch({
+            type: 'ERROR',
+            payload: err.message
+          })
+        else
+          dispatch({
+            type: 'ERROR',
+            payload: err.response.data.message[0].messages[0].message
```
Comment (kbhutani0001) [inline src/hooks/useAuth.js:122]: can we have some null checks here. This is going to throw a lot of errors if any of them goes undefined

Comment (kbhutani0001) [inline src/hooks/useAuth.js:122]:
```suggestion
            payload: err.response.data?.message?.[0]?.messages?.[0]?.message
```

Comment (SundeepChand) [reply src/hooks/useAuth.js:122]: Should I commit the suggested changes?

Comment (kbhutani0001) [reply src/hooks/useAuth.js:122]: yes, you can

**Pull request merged**